python_modules/dagster/dagster_tests/asset_defs_tests/partition_mapping_tests/test_time_window_partition_mapping.py

```diff
@@ -727,3 +727,33 @@ def test_mar_2024_dst_transition_with_hourly_partitions():
     assert downstream.get_partition_keys(current_time=current_time) == [
         "2024-03-10-04:00",
     ]
+
+
+def test_nov_2023_dst_transition_with_hourly_partitions():
+    partitions_def = HourlyPartitionsDefinition("2023-11-01-00:00", timezone="America/Los_Angeles")
+    time_partition_mapping = TimeWindowPartitionMapping(start_offset=-1, end_offset=-1)
+    current_time = datetime(2023, 11, 5, 10)
+
+    subset = partitions_def.subset_with_partition_keys(["2023-11-05-01:00"])
+    upstream = time_partition_mapping.get_upstream_mapped_partitions_result_for_partitions(
```

Comment:
think it would be good to also test the downstream direction

Reply:
good call, added